services/src/main/java/org/keycloak/services/resources/WelcomeResource.java

```diff
@@ -189,16 +189,20 @@ private Response createWelcomePage(String successMessage, String errorMessage) {
                 return Response.status(302).location(adminUrl).build();
             }
 
-            Map<String, Object> map = new HashMap<>();
+            String commonPath = themeProperties.getProperty("common");
+            if (commonPath == null || commonPath.isEmpty()) {
+                commonPath = "common/keycloak";
+            }
```

Comment:
I believe we can provide a default value to `getProperty()` here, it wouldn't handle the empty case, but that is a user-error anyways.
```suggestion
            String commonPath = themeProperties.getProperty("common", "common/keycloak");
```